homeassistant/components/shell_command.py

```diff
@@ -62,6 +62,8 @@ def async_service_handler(service: ServiceCall) -> None:
 
         if rendered_args == args:
             # No template used. default behavior
+
+            # pylint: disable=no-member
             create_process = asyncio.subprocess.create_subprocess_shell(
```

Comment:
I can't seem to find this method in the docs for Python 3.4 https://docs.python.org/3.4/library/asyncio-subprocess.html

I do see `asyncio.create_subprocess_shell`

Reply:
Seems to import fine on 3.4.2 for me (which I think I kept around because it was default on Jessie IIRC).

Docs do make it look weird, but it’s there.

https://github.com/python/cpython/blob/3.4/Lib/asyncio/subprocess.py

Reply:
Ok, sounds good 👍